tests/helpers/cluster.ts

```diff
@@ -11,6 +11,10 @@ export const skipIfNotSupported = (lxdVersion: LxdVersions) => {
   );
 };
 
+export const skipIfNotClustered = (projectName: string) => {
+  test.skip(projectName.includes("clustered"));
```

Comment:
```suggestion
  test.skip(projectName.includes("-clustered"));
```
Without the prefix, `unclustered` matches as well.